build-system/sauce_connect/OWNERS

```diff
@@ -0,0 +1,13 @@
+// For an explanation of the OWNERS rules and syntax, see:
+// https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example
+
+{
+  rules: [
+    {
+      owners: [
+        { name: 'ampproject/wg-infra' },
+        { name: 'rsimha', notify: true }
```

Comment:
You can add me here as a notify too

Reply:
Already merged. Can you add in a new PR?